app/components/primer/box_component.rb

```diff
@@ -18,5 +18,9 @@ def initialize(**system_arguments)
     def call
       render(Primer::BaseComponent.new(**@system_arguments)) { content }
     end
+
+    def self.status
```

Comment:
FWIW, I think we could achieve this via YARD as well, perhaps via the `version` tag?

See https://www.rubydoc.info/gems/yard/file/docs/Tags.md#adding-custom-tags.

That being said, I think it's cool to have this status available programmatically 👍🏻

Reply:
I started with `tags` but got confused ;). Happy to revisit this if it's better / easier.

Reply:
🤷🏻‍♂️ I think they're equivalent at this point. Just wanted to note this in case we have any issues with this approach.